### ISSUE TITLE
"Will THIS break something"

### ISSUE THREAD

**Issue by kerelape** [elidhan/Simple-Animated-Guns#19]
https://github.com/elidhan/Simple-Animated-Guns/blob/5cf8a0cbd59efb6004a4a629a911430607b1cc96/src/main/java/net/elidhan/anim_guns/client/render/GunRenderer.java#L57C37-L57C37

No.

... at least shouldn't

**Comment by windy664** [comment]
Indeed, it will ignore the permissions of ftbchunks and directly destroy the glass
